### PR TITLE
Enhancement/tangipahoa so

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: clean/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            "name": "Python: Current File (Integrated Terminal)",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "cwd": "${workspaceRoot}",
+            "env": {"PYTHONPATH": "${workspaceRoot}"},
+            "console": "integratedTerminal"
+        }
+    ]
+}
```
Comment (pckhoi) [inline clean/.vscode/launch.json:17]: Please remove this file

File: data/.wrglsync.yaml [hunk 2]
```diff
@@ -318,9 +324,15 @@ repos:
...
       - event_uid
+<<<<<<< HEAD
+  event-tangipahoa-so:
+    fileName: fuse/event_tangipahoa_so.csv
+    primaryKey: 
+=======
   event-new-orleans-so:
     fileName: fuse/event_new_orleans_so.csv
     primaryKey:
+>>>>>>> 9240ec865f6bf2d1002ee37427730d551439f56e
```
Comment (pckhoi) [inline data/.wrglsync.yaml:335]: Looks like this merge conflict is left unresolved. Please read [this](https://git-scm.com/docs/git-merge#_how_conflicts_are_presented) to understand more about how conflicts are presented.

Comment (ayyubibrahimi) [reply data/.wrglsync.yaml]: Updated.

File: clean/.vscode/launch.json (hunk 1 above)
Comment (ayyubibrahimi) [reply clean/.vscode/launch.json]: Removed.

**Review by pckhoi — APPROVED** [review]
LGTM

**Pull request merged**